source/features/comments-time-machine-links.tsx

```diff
@@ -117,9 +116,17 @@ async function init(signal: AbortSignal): Promise<void> {
 			.datetime
 			.value;
 
-		addInlineLinks(menu, timestamp);
+		addInlineLinks(menu.closest('.js-comment')!, timestamp);
 		addDropdownLink(menu, timestamp);
 	}, {signal});
+
+	observe([
+		'div.react-issue-comment',
+		'[data-testid="review-thread"] > div',
+	], comment => {
+		const timestamp = comment.querySelector('relative-time')!.attributes.datetime.value;
```

Comment:
```suggestion
		const timestamp = $('relative-time', comment).attributes.datetime.value;
```

Reply:
😂 Its generated by copilot. I dont check it. My bad.